Apps/Examples/Examples/All Examples/FeatureStateExample.swift

```diff
@@ -0,0 +1,321 @@
+import UIKit
+import MapboxMaps
+
+@objc(FeatureStateExample)
+
```

Comment:
```suggestion
```

Let's remove this empty line to make it clear that the annotation is attached to the class declaration